### ISSUE TITLE
Check all notebooks/examples are working after renaming

### ISSUE THREAD

**Issue by lgvaz** [airctic/icevision#380]
Tutorials:
- [ ] custom_parser
- [x] dataset_voc_nb
- [x] efficientdet_pets
- [ ] getting_started
- [ ] how_to_train_dataset
- [ ] inference
- [x] mask_rcnn_pennfundan

**Comment by adamfarquhar** [comment]
I've done quite a bit of editing on the getting started tutorial. There is still some work to do before checking it in.

- Suggest merging 5.2 and 5.3 as the split is really part of parse. The splitter doesn't actually split.
- Suggest setting the seed at the top to make sure that the demo is completely repeatable. Which seed?
- **Need** comment on presize vs size. I can't do this, as I don't know what presize is!
- **Need** clarification: 'Because we normalized our images with imagenet_stats' - not in evidence! Where is this happening?
- The example model requires 3m/epoch for 10 epochs, or 30min on the free colab (at least for me). I think that this is too long for a tutorial.
- **Need** clarification:  'Training the model with fastai using fine_tune twice and I got led the the following results'. I don't understand this sentence.

**Comment by adamfarquhar** [comment]
In getting started, the weights_url seems to be broken:

![image](https://user-images.githubusercontent.com/454453/92591187-534fa980-f295-11ea-9abf-0e036aa7293c.png)

**Comment by adamfarquhar** [comment]
Getting_started -
This currently ends with prediction. I wonder if it might make sense to reverse the order (like the detectron2 tutorial) and have prediction before training. The pets training data is pretty slow to train (30m on free colab gpu) and done once for fastai and once for PL. That means an hour of sitting watching progress bars, which is not a great user experience.

**Comment by ai-fast-track** [comment]
Thank you Adam. I just updated the weights urls in different notebooks in the PR #395.

With regard to the order (Inference then training), we do have a notebook that shows the inference first in the [Inference NB](https://airctic.com/inference/).

**Comment by lgvaz** [comment]
Hello @adamfarquhar, sorry for the delay in the response, I initially thought that all the questions you asked were addressed in our chat via Discord, but after re-reading your questions I'm not sure everything was clarified.

Does any of those points still need clarification?
